applications/StructuralMechanicsApplication/custom_constitutive/linear_plane_stress.h

```diff
@@ -17,18 +17,34 @@
 // External includes
 
 // Project includes
-#include "includes/constitutive_law.h"
+#include "custom_constitutive/elastic_isotropic_3d.h"
 
 namespace Kratos
 {
+///@name Kratos Globals
+///@{
 
+///@}
+///@name Type Definitions
+///@{
 
-class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) LinearPlaneStress : public ConstitutiveLaw
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION) LinearPlaneStress : public ElasticIsotropic3D
```

Comment:
Such a change should be discussed first!
I see that this is done with come other laws but I have to check in more detail first since 
@RiccardoRossi @loumalouomega 

furthermore if you change the BaseClass, you also have to update the serializer functions